FEXCore/include/FEXCore/Utils/AllocatorHooks.h

```diff
@@ -138,8 +134,8 @@ inline bool VirtualProtect(void* Ptr, size_t Size, ProtectOptions options) {
 
 #endif
 
-// Memory allocation routines aliased to jemalloc functions.
-#ifdef ENABLE_JEMALLOC
+#if !defined(_WIN32)
```

Comment:
Need to revisit this. Windows-like targets now pull in the JemallocDummy, which is probably fine, but the dummy's .cpp file should adopt appropriate ifdefs.

Reply:
Windows targets should use actual jemalloc, malloc calls into it (Common/CRT)

Reply:
That being said, redirecting malloc to je_malloc to host malloc is really confusing, surely building a FEXCore target that uses  the system malloc directly would be simpler than this, or perhaps just split out allocatorhooks entirely and expect each static FEXCore user to provide it's own impl?

Reply:
> Windows targets should use actual jemalloc, malloc calls into it (Common/CRT)

Oof, so previously FEXCore::Allocator::malloc actually called Common/CRT's malloc, which then called into jemalloc?

> That being said, redirecting malloc to je_malloc to host malloc is really confusing

True... previously these headers have been restructured specifically to avoid one pointer indirection per call, but that seems negligible overall. I'll look into making JemallocDummy override malloc & friends instead.

Reply:
> > Windows targets should use actual jemalloc, malloc calls into it (Common/CRT)
> 
> Oof, so previously FEXCore::Allocator::malloc actually called Common/CRT's malloc, which then called into jemalloc?
> 
That would be the case for actual malloc calls from e.g. libc++ startup but because windows defines ENABLE_JEMALLOC FEXCore::Allocator::malloc would call directly into jemalloc skipping the crt in this case